intl/messages/br.json

```diff
@@ -0,0 +1,48 @@
+    "footerRightContent": "IPFS.JS Lorem ipsum",
+    "whatCanYouBuildTitle": "Lorem ipsum",
+    "whatCanYouBuildSectionDesc": "Nullam quis risus eget  urna mollis ornare vel eu leo. Maecenas sed diam eget risus varius magna. Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    "whatCanYouBuildItem1Name": "PT-Video Streaming",
+    "whatCanYouBuildItem1Title": "PT-Mornon mi porta gravida",
+    "whatCanYouBuildItem1Subtitle": "PT-Maecenas faucibus mollis interd ide linon mi porta gravida eget metus.",
+    "whatCanYouBuildItem1Desc": "PT-Nullam id dolor id nib ultricies vehicula ut id elit. Commodo cursus magna, vel sce i sque nisl constur et. Nullam id dolor id nib ultricies vehicula ut id elit. Commodo cursus magna, vel scei sque nisl consectetur et.",
+    "whatCanYouBuildItem2Name": "PT-Real Time Applications",
+    "whatCanYouBuildItem2Title": "PT-Mornon mi porta gravida",
+    "whatCanYouBuildItem2Subtitle": "PT-Maecenas faucibus mollis interd ide linon mi porta gravida eget metus.",
+    "whatCanYouBuildItem2Desc": "PT-Nullam id dolor id nib ultricies vehicula ut id elit. Commodo cursus magna, vel sce i sque nisl constur et. Nullam id dolor id nib ultricies vehicula ut id elit. Commodo cursus magna, vel scei sque nisl consectetur et.",
+    "whatCanYouBuildItem3Name": "PT-Real Time Applications",
+    "whatCanYouBuildItem3Title": "PT-Mornon mi porta gravida",
+    "whatCanYouBuildItem3Subtitle": "PT-Maecenas faucibus mollis interd ide linon mi porta gravida eget metus.",
+    "whatCanYouBuildItem3Desc": "PT-Nullam id dolor id nib ultricies vehicula ut id elit. Commodo cursus magna, vel sce i sque nisl constur et. Nullam id dolor id nib ultricies vehicula ut id elit. Commodo cursus magna, vel scei sque nisl consectetur et.",
+    "whatPeopleAreBuildingTitle": "Lorem ipsum",
+    "whatPeopleAreBuildingSectionDesc": "PT-Maecenas sed eu leo. s eget urna mollis ornare vel eu leo diam eget risus varius blaecenas sed diam eget risus varius blandit sit amet non magna. Lorem ipsum dolor sit amet.",
+    "whatArePeopleBuildingProj1Desc": "PT-Maecenas faucibus mollis interd. Donec id elit non mi porta gravida eget metus. Nullam id dolor id nib ultricies vehicula ut id elit.",
+    "publicationsAndTalksTitle": "PT-Publications & talks",
+    "publicationsAndTalksSectionDesc": "PT-Nullam quis risus eget  urna mollis ornare vel eu leo. Maecenas sed diam eget risus varius magna. Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    "publicationAndTalk1Desc": "PT-Nullam id dolor id nib ultricies vehicula ut id elit.",
+    "publicationAndTalk2Desc": "PT-Maecenas faucibus mollis interd. Donec id elit non mi porta gravida eget metus.",
+    "publicationAndTalk3Desc": "PT-Maecenas faucibus mollis iterdum. Curabitur blandit tempus porttitor.",
+    "publicationAndTalk4Desc": "PT-Nullam id dolor id elit non mi porta mollis ornare vel eu leo.",
+    "service-worker": "Iniciar Service Worker"
+}
```

Comment:
@PedroMiguelSS upgrade the org of this doc to be separated by sections. Example:

```
{
  nav: {
    // nav stuff
  },
  hero: {
    // hero stuff
  },
  features: {
    list: [
      {
        title: "asjdka",
        description: "asjdka"
      },
      {
        title: "asjdka",
        description: "asjdka"
      },
    ]
  }
  // and so on
}
```